### PR TITLE
Add Docker Compose for integration tests

### PR DESCRIPTION
**Pull request by AlekSi**
/cc @SuperQ

### CODE REVIEW

**Comment by SuperQ** [comment]
Very cool!  Thanks!

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,29 @@
-sudo: false
+dist: trusty
```
Comment (SuperQ) [inline .travis.yml]: Any reason we shouldn't use `xenial` or `stretch` here?

File: .travis.yml [hunk 2]
```diff
@@ -1,11 +1,29 @@
...
+sudo: required
+language: go
```
Comment (SuperQ) [inline .travis.yml]: This seems like a dupe of Line 5.

File: .travis.yml (hunk 1 above)
Comment (AlekSi) [reply .travis.yml]: Travis CI doesn't have them: https://docs.travis-ci.com/user/reference/overview/

File: .travis.yml (hunk 2 above)
Comment (AlekSi) [reply .travis.yml]: Right, will fix it.

Comment (AlekSi) [reply .travis.yml]: Done.

File: .travis.yml (hunk 1 above)
Comment (SuperQ) [reply .travis.yml]: Wow, ok, more reason to start moving Prometheus over to our new buildkite account. :-)

**Pull request merged**